packages/components/src/components/hds/modal/index.ts

```diff
@@ -9,17 +9,57 @@ import { action } from '@ember/object';
 import { assert } from '@ember/debug';
 import { getElementId } from '../../../utils/hds-get-element-id.ts';
 import { buildWaiter } from '@ember/test-waiters';
+import type { WithBoundArgs } from '@glint/template';
+
+import HdsDialogPrimitiveHeaderComponent from '../dialog-primitive/header.ts';
+import HdsDialogPrimitiveBodyComponent from '../dialog-primitive/body.ts';
+import HdsDialogPrimitiveFooterComponent from '../dialog-primitive/footer.ts';
+import type { HdsModalSizes, HdsModalColors } from './types.ts';
+import { HdsModalSizeValues, HdsModalColorValues } from './types.ts';
```

Comment:
nit: we tend to group imports by type
```suggestion
import type { WithBoundArgs } from '@glint/template';
import type { HdsModalSizes, HdsModalColors } from './types.ts';

import HdsDialogPrimitiveHeaderComponent from '../dialog-primitive/header.ts';
import HdsDialogPrimitiveBodyComponent from '../dialog-primitive/body.ts';
import HdsDialogPrimitiveFooterComponent from '../dialog-primitive/footer.ts';
import { HdsModalSizeValues, HdsModalColorValues } from './types.ts';
```